### DISCUSSION TITLE
Bulk Downloader for Reddit 2.2

### DISCUSSION THREAD

**Discussion by aliparlakci** [Serene-Arc/bulk-downloader-for-reddit#441] [General]
## Changelog
- Add `clone` mode #420 
- Add `--disable-module` option #341 
- Includes submission date in file metadata #387 
- Fetches more information in `archive` mode #408 
- Makes hash calculation more efficient on RAM #416
- Fixes path length issue #385 
- Fixes file extension bug in Imgur sources #403 
- Fixes emoji issue on macos #407 
- Minor changes and bug fixes. See commit history for more information.

<hr /><em>This discussion was created from the release <a href='https://github.com/aliparlakci/bulk-downloader-for-reddit/releases/tag/v.2.2.0'>Bulk Downloader for Reddit 2.2</a>.</em>

**Comment by Null42x86** [comment]
Very good, how do i run this thing? is because i'm a newbie here
